docs/api/turn-by-turn/api-reference.md

```diff
@@ -2,7 +2,7 @@
 
 Valhalla's routing service (a.k.a. turn-by-turn), is an open-source routing service that lets you integrate routing and navigation into a web or mobile application.
 
-[View an interactive demo](http://valhalla.github.io/demos/routing)
+[View an interactive demo](https://valhalla.openstreetmap.de)
```

Comment:
i dont know im iffy on these types of changes. i think there is room for both types of demos. the programs we have have in the demos repo are for developers who are running their own valhalla and want to play with it. the demo at valhalla.openstreetmap.de is more for high level people who want to just see vaguely how it works. i'm happy to have both linked but i dont want to lose visibility of the tools we have in the demos repo. i personally think linking to the new interactive demos that are hosted makes the most sense on the main README because that is where people who are only taking a shallower read and who would appreciate a front end would tend to go.

basically i dont want to lose the links to the existing demos and tools even if they require you to be running your own service and are terrible as UI goes hahahha, not saying we cant put in the link to the new stuff, just dont want to lose the old stuff as i think it still serves a purpose (debugging for developers)

how can we serve both use-cases?

Reply:
no worries, I think you're right. this is more the area where a dev is roaming around, makes sense to link more of a dev tool. I removed those changes. that reminds me though that the demos could be updated to hit the open server rather than the key-protected current server to work out-of-the-box, right?